memorypipeline/appsettings.json

```diff
@@ -57,23 +57,23 @@
       // - Directory is the location where files are stored.
       //
       "SimpleFileStorage": {
-        "Directory": "../webapi/tmp-cache"
+        "Directory": "../tmp/cache"
```

Comment:
We should also update the gitignore file

Reply:
Great call...doing now!